docker/install-scripts/ansible.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+while [ $# -gt 0 ]; do
+  case "$1" in
+    --version=*)
+      VERSION="${1#*=}"
+      ;;
+    *)
+      printf "***************************\n"
```

Comment:
I dont think there is a need to do this type of log formatting.

Reply:
fixed